docs/explanations/curator_data_model.md

```diff
@@ -0,0 +1,347 @@
+# CSV data model description
+
+The Curator-Extension (formerly Schematic) data model is used to create JSON Schemas for Curator. See [JSON Schema documentation](https://json-schema.org/). This is used for the DCCs that prefer working in a tabular format (CSV) over JSON or LinkML. A data model is created in the format specified below. Then the Curator-Extension in the Synapse Python Client can be used to convert to JSON Schema.
+
+A link will be provided here to documentation for converting CSV data models to JSON Schema in the near future.
+
+## Data model columns
+
+A JSON Schema is made up of one data type(for example a person) and the attributes that describe the data type (for example age and gender). The CSV data model will describe one or more data types. Each row describes either a data type, or an attribute.
+
+Data types:
+
+- must have a unique name in the `Attribute` column
+- must have at least one attribute in the `DependsOn` column
+- may have a value in the `Description` column
+
+Attributes:
+
+- must have a unique name in the `Attribute` column
+- may have all values  for all other columns besides `DependsOn`
+
+The following data model has one data type, `Person`, and that data type has one attribute, `Gender`.
+
+| Attribute | DependsOn |
+|---|---|
+| Person    | "Gender"  |
+| Gender    |           |
+
+Converting the above data model to JSON Schema results in:
+
+```json
+{
+  "description": "TBD",
+  "properties": {
+    "Gender": {
+      "description": "TBD",
+      "title": "Gender"
+    }
+  }
+}
+```
+
+### Attribute
+
+The name of the data type or attribute being described on this line. This should be a unique identifier in the file. For attributes this will be translated as the title in the JSON Schema.
+
+### DependsOn
+
+The set of of attributes this data type has. These must be attributes that exists in this data model. Each attribute will appear in the properties of the JSON Schema. This should be a comma-separated list in quotes. Example: "Patient ID, Sex, Year of Birth, Diagnosis"
```

Comment:
I think here you switch the context from describing "Person" in the previous example to "Patient". It will be easier for readers to just stick to the same data type.